src/clarity.rs

```diff
@@ -200,6 +207,44 @@ where F: FnOnce(MarfedKV) -> (MarfedKV, R) {
     result
 }
 
+fn eval(invoked_by: &str, args: &[String]) -> EvalInput {
+    if args.len() < 3 {
+        eprintln!("Usage: {} {} [contract-identifier] (program.clar) [vm-state.db]", invoked_by, args[0]);
+        panic_test!();
+    }
+
+    let vm_filename = 
+        if args.len() == 3 {
+            &args[2]
+        } else {
+            &args[3]
```

Comment:
This can panic with out-of-bounds, since the `args.len() < 3` check above will allow `args` to have 3 items.

Reply:
This was basically just a copy/paste of the previous code. So, should the args length check only allow 3 or 4 entries, otherwise panic?

Reply:
I updated the args.len() check to only allow length of 3 or 4.

Reply:
Thanks!